Add refresh command

stefhol opened the pull request:
Hi its me again :)
The command generates new tmux worktree windows in a session.
so 
```
tms refresh
```
Wil create missing worktrees windows in the current directory.
```
tms refresh session-name 
```
Will create missing worktree windows for the session `session-name`

https://github.com/jrmoulton/tmux-sessionizer/assets/35531567/6036fb8e-22b3-4225-a214-cc07836ab1fd

Note: This differs a bit from the current implementation as I don't check if the repository is bare. 
In my opinion the user should be able to decide if he wants to use bare with worktree or normal with worktree. (I am such a user that uses both) and I would argue that someone who uses the git feature has made his mind up.
So probably provide a config option or change the default behaviour. 

Currently the implementation is kind of a duplicate to the startup one. So further work should be to extract the generation of windows in a utility function that is called on refresh call and on first session generation. 

Cheers

jrmoulton commented:
Awesome! I've been warning this for a long time. 

This looks great to me

Can you update the README with the updated output from the help command?

stefhol commented:
I added the Readme. I also found some bugs with prunable worktrees and I added creation of a base repository window if the respoistory has only worktree windows and is not bare

jrmoulton commented:
LGTM. Thanks!
